MongoTrait/SoftDeleteable.php

```diff
@@ -11,6 +11,7 @@
      * @var boolean
      *
      * @ODM\Field(type="boolean")
+     * @ORCHESTRA\Search(key="deleted", type="boolean")
```

Comment:
This attributes is not displayed, but we can add the search parameter on it (for the site for instance)